pants

```diff
@@ -11,6 +11,13 @@ set -eo pipefail
 
 HERE=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
 
+# Source any custom bootstrap settings for Pants from PANTS_BOOTSTRAP if it exists.
+: "${PANTS_BOOTSTRAP:=.pants.bootstrap}"
```

Comment:
?

Reply:
Is this intentionally part of this PR? Seems like it should be separate? And was this updated in pantsbuild/setup? Or is this intended to be just in this repo?

Reply:
OTOH, it's completely internal, so does it really matter?